samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,47 @@
+Highcharts.chart('container', {
+    title: {
+        text: 'Global temperature mean anomalies from 1880 to 2016',
+        align: 'left'
+    },
+    accessibility: {
+        description: `
+            A chart of monthly mean temperature anomalies from 1880 to 2016.
+            It employs two series, one column and one line,
+            to convey the same data at different resolutions.
+            There is only one column per thirty years,
+            but the line series have a datapoint per year.
+        `
+    },
+    series: [{
+        type: 'column',
+        name: 'Average per thirty years',
+        data: JSON.parse(document.getElementById('columnData').textContent)
```

Comment:
The column series should also have a tooltip format to reflect its time span. Use a format string to render something like `1990-2020`. Use the built-in `subtract` function of the Templating.

Reply:
In order to subtract 30 years from a timestamp its necessary to do some calculations. It does function now, however the operation looks a bit messy:

```
tooltip: {
    headerFormat: '<span>{point.x:%Y} ' +
        '{(subtract point.x (multiply 30 ' +
        '(multiply 60 (multiply 1000' +
        '(multiply 60 (multiply 365.25 24)))))):%Y}</span>'
}
```

Can and should we use custom variables in templating to make this more readable?

Reply:
I think we can just use the number of milliseconds and compensate with a comment:
```js
            // Subtract 30 years from the year to get the start of the period
            headerFormat:
                '<span>{(subtract point.x 946684800000):%Y} - {point.x:%Y}<br>'
```